### ISSUE TITLE
Expose fork name in api

### ISSUE THREAD

**Issue by Aldarone** [glitch-soc/mastodon#1592]
### Pitch

Modify the version string exposed by the API to include the fork's name in a sensible way.

- Currently : `3.2.0`
- Proposal : `3.2.0 (compatible; glitch-soc)`

### Motivation

Mastodon forks and MastoAPI compatible software often have additional features not supported by mastodon.

Those features are generally only available through the web interface because third party clients act as if they're interacting with vanilla Mastodon.

Exposing the fork's name can help them doing backend feature detection.

**Comment by ClearlyClaire** [comment]
The fork name is already included in the version, it's currently `3.4.1+glitch`. In the future, specific feature flags are probably going to be in `/api/v1/instance`, which currently includes a `configuration` entry for Mastodon and glitch-soc alike:

```json
  "configuration": {
    "statuses": {
      "max_characters": 500,
      "max_media_attachments": 4,
      "characters_reserved_per_url": 23
    },
    "media_attachments": {
      "supported_mime_types": [
        "image/jpeg",
        "image/png",
        "image/gif",
        "video/webm",
        "video/mp4",
        "video/quicktime",
        "video/ogg",
        "audio/wave",
        "audio/wav",
        "audio/x-wav",
        "audio/x-pn-wave",
        "audio/ogg",
        "audio/mpeg",
        "audio/mp3",
        "audio/webm",
        "audio/flac",
        "audio/aac",
        "audio/m4a",
        "audio/x-m4a",
        "audio/mp4",
        "audio/3gpp",
        "video/x-ms-asf"
      ],
      "image_size_limit": 10485760,
      "image_matrix_limit": 16777216,
      "video_size_limit": 41943040,
      "video_frame_rate_limit": 60,
      "video_matrix_limit": 2304000
    },
    "polls": {
      "max_options": 4,
      "max_characters_per_option": 50,
      "min_expiration": 300,
      "max_expiration": 2629746
    }
  },

```

**Comment by Aldarone** [comment]
My bad, the instance I checked was vanilla… Nevermind then :sweat_smile:
